src/main/java/com/vaadin/training/bugrap/ui/BugrapApplicationModel.java

```diff
@@ -68,4 +84,55 @@ public boolean isAllVersionsSelected() {
 		return projectVersion == null;
 	}
 
+	public String getElapsedTimeFrom(final Date timestamp) {
```

Comment:
This could be perhaps refactored into own utility class, since the conversion from date to string doesn't have state involved